shippo/test/integration/test_integration.py

```diff
@@ -62,17 +62,6 @@ def test_list_accessors(self):
         address['foo'] = 'bar'
         self.assertEqual(address.foo, 'bar')
 
-    def test_raise(self):
```

Comment:
We are not raising exceptions on those cases anymore. Just marking them as imcomplete.

This test was really redundant to one in `test_address`. So I removed it